docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -33,6 +33,14 @@ The main features of the Media Controller API include:
 
   You can [set age rating for currently playing media](#setting_age_rating) by using updatePlaybackAgeRating() server method.
 
+- Managing playlists on server side
+
+  You can [manage playlists](#managing-playlists-on-server-side) on server side by creating, saving, deleting and getting information about playlists. You can also update the playback item from the playlists item list.
```

Comment:
Please add a serial comma before and:

.... saving, deleting, and getting.....

Reply:
Done